Formula/geesefs.rb

```diff
@@ -0,0 +1,24 @@
+class Geesefs < Formula
+  desc "Finally, a good FUSE FS implementation over S3"
+  homepage "https://github.com/yandex-cloud/geesefs"
+  url "https://github.com/yandex-cloud/geesefs/archive/refs/heads/master.tar.gz"
```

Comment:
We don't build from `master`. Please use a tarball generated from a tag.

Reply:
Of course, it was just for testing. Sorry for sending this change :-)